opencog/scm/opencog/rule-engine/rule-engine-utils.scm

```diff
@@ -313,6 +314,34 @@
 "
   (map (lambda (x) (gen-rand-variable prefix base length)) (iota n)))
 
+(define (set-ure-logger-level! level)
```

Comment:
I think function names like `ure-logger-set-level!` would be better as they would be more consistent with the existing function names. What do you think?